### DISCUSSION TITLE
Bug report 「缺陷报告」apt build-dep does not install full dependency for deepin-camera

### DISCUSSION THREAD

**Discussion by akr2002** [linuxdeepin/developer-center#2864] [Q&A | 问答板块] [answered]
## Missing dependencies
`libgtest-dev` `libgmock-dev`

**Comment by zccrs** [comment]
Thank you.

**Comment by wineee** [comment, marked as answer]
These packages are only used for testing，I  think they shouldn't add  as necessary dependencies

Use `Release` mode instead of `Debug` ，unless you really need to debug
> cmake -DCMAKE_BUILD_TYPE=Release

**Comment by akr2002** [comment]
Ah fair
